charts/asset-manager/templates/worker-deployment.yaml

```diff
@@ -69,6 +69,9 @@ spec:
           volumeMounts:
             - name: clam-virus-db
               mountPath: /var/lib/clamav
+            - name: {{ .Release.Name }}-clamd-conf
```

Comment:
No need for `Release.Name` here. The name is scoped to within the pod.

Reply:
I can fix that, thanks